internal/services/serviceconnector/sdk/2022-05-01/links/client.go

```diff
@@ -0,0 +1,15 @@
+package links
```

Comment:
Where is this sdk comming from? we should only be using the go-azure-sdk one?

Reply:
This is the old sdk, removed.